src/campaigns/swapsPromoCampaign.ts

```diff
@@ -48,31 +48,36 @@ export const swapsCampaignAction = async () => {
 export enum SwapsPromoCampaignExclusion {
   noAssets = 'no_assets',
   alreadySwapped = 'already_swapped',
-  firstLaunch = 'first_launch',
   wrongNetwork = 'wrong_network',
 }
 
 export const swapsCampaignCheck = async (): Promise<
   SwapsPromoCampaignExclusion | GenericCampaignCheckResponse
 > => {
   const hasShownCampaign = mmkv.getBoolean(CampaignKey.swapsLaunch);
-
-  // we only want to show this campaign once
-  if (hasShownCampaign) return GenericCampaignCheckResponse.nonstarter;
+  const isFirstLaunch = mmkv.getBoolean(STORAGE_IDS.FIRST_APP_LAUNCH);
 
   const {
     selected: currentWallet,
   }: {
     selected: RainbowWallet | undefined;
   } = store.getState().wallets;
 
-  // if there's no wallet or the current wallet is read only then stop
-  if (!currentWallet || currentWallet.type === WalletTypes.readOnly)
+  /**
+   * stop if:
+   * there's no wallet
+   * the current wallet is read only
+   * the campaign has already been activated
+   * the user is launching Rainbow for the first time
+   */
+  if (
+    !currentWallet ||
+    currentWallet.type === WalletTypes.readOnly ||
```

Comment:
How much do we want to change this since it's already in prod? Also, for read-only, we could still let the user know, we just need to remove the main CTA or something.

Reply:
this whole change seems unneeded, we should use cards or something less in the users face if the feature isnt immediately actionable imo

Reply:
i dont think the logic is any different with this change? i just grouped together 3 separate checks that all result in the same thing, aka `GenericCampaignCheckResponse.nonstarter`

Reply:
the only actual difference is that with this change we would no longer be tracking in analytics the users who didn't qualify for the campaign bc it was their first time launching rainbow. to me that seemed an extraneous detail.